bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/JsonUtils.java

```diff
@@ -179,19 +176,22 @@ private static Object getMappingElement(Object json, BString elementName, boolea
                 return null;
             }
 
-            return ErrorCreator.createError(MAP_KEY_NOT_FOUND_ERROR, StringUtils
-                    .fromString("Key '" + elementName + "' not found in JSON mapping"));
+            return ErrorUtils.getRuntimeError(BALLERINA_LANG_MAP_PKG_ID,
```

Comment:
Can we format this to one line?

Reply:
If I kept it in one line, the line length will exceed 120 characters (122 characters are there)